src/pairselection.cxx

```diff
@@ -838,7 +840,8 @@ ROOT::RDF::RNode PairSelection(ROOT::RDF::RNode df,
     Logger::get("tautau::PairSelection")
         ->debug("Setting up TauTau pair building");
     auto df1 = df.Define(
-        pairname, pairselection::fullhadronic::PairSelectionAlgo(mindeltaR),
+        pairname,
+        ditau_pairselection::fullhadronic::PairSelectionAlgo(mindeltaR),
```

Comment:
shall we name this function fullyhadronic?

Reply:
I would keep it as fullhadronic :)